### DISCUSSION TITLE
Can I record videos with Camposer while using the imageAnalyzer?

### DISCUSSION THREAD

**Discussion by RaulETSINF** [ujizin/Camposer#23] [Q&A] [answered]
If it is possible, how??

**Comment by ujizin** [comment]
Hey @RaulETSINF, my apologies for the delay. 

Currently, it is not possible to record videos with Camposer while using the imageAnalyzer.  😢 

I'll work on developing this feature ASAP. Please keep in mind that even when it's implemented, this capability might not be available on all devices. 

You can find more information about this [here](https://youtu.be/fazzQs-O31U?t=101)

Thank you for opening the discussion and understanding.

**Comment by ujizin** [comment, marked as answer]
Hello @RaulETSINF, it's now possible to record videos using the imageAnalyzer in version 0.3.0 😄 

Thank you once again for initiating the discussion!

**Reply by RaulETSINF** [reply]
Awesome Work!! Cool! :). Thanks for updating.
